features/step_definitions/login_steps.rb

```diff
@@ -48,17 +26,17 @@
   }
 end
 
-Given 'I have a buyer' do
-  @buyer = step "I have a buyer user"
+Given 'I have a buyer user' do
+  @buyer_user = create_user('buyer')
 end
 
-Given 'I have a supplier' do
-  @supplier = create_supplier
-  @supplier_user = step "I have a supplier user with supplier id #{@supplier['id']}"
+Given 'I have a supplier user' do
+  @supplier = create_supplier()
```

Comment:
What does adding the parentheses here change?  Nothing at all? Does Ruby execute the method even without the brackets?

Reply:
It's nothing at all...
https://stackoverflow.com/a/342475 
My 🐍  sense is deeply unsettled.

Reply:
Removed redundant parenthesis